### ISSUE TITLE
not working on 1.16.5

### ISSUE THREAD

**Issue by detecjack** [way2muchnoise/JustEnoughResources#292]

**Comment by XxZajoZzO** [comment]
Same here, nothing shows up in recipes.

**Comment by way2muchnoise** [comment]
Could you test with one of the new builds target at 1.16.5 on CF?

https://www.curseforge.com/minecraft/mc-mods/just-enough-resources-jer/files/all?filter-status=1&filter-game-version=2020709689%3A8203

**Comment by Peca21** [comment]
On my survival world it's still not working. Showing recipes for iron ore doesn't give ne the generation panel. I am on JustEnoughResources-1.16.5-0.12.1.119

**Comment by set92** [comment]
I modified a Valhelsia 3 server and added JustEnoughResources-1.16.5-0.12.1.121 but even when it appears in the main menu - mods, when I try to check the generation panel on iron ore doesn't appear, I also tried /jer_profile is not recognized.

**Comment by 04Slash** [comment]
FWIW, everything seems to be working just fine using `jei-1.16.5-7.7.0.104` and `JustEnoughResources-1.16.5-0.12.1.121` in single player. Just looking around for a few minutes, Mob Drops, Plant Drops, Dungeon Loot, Villager Trades, Ore Gen, all seem to be working just fine. [Here's an album](https://imgur.com/a/HAPUDgo) showing the above working on the above listed mod versions. Initially tested on Forge 36.1.4 and confirmed still working on 36.1.25 (in case this makes a difference).

**Comment by Peca21** [comment]
Still doesn't work on my modpack/world. 
jei-1.16.5-7.7.0.104
JustEnoughResources-1.16.5-0.12.1.121
.25 forge

**Comment by 04Slash** [comment]
> Still doesn't work on my modpack/world.
> jei-1.16.5-7.7.0.104
> JustEnoughResources-1.16.5-0.12.1.121
> .25 forge

Does it work in a modpack with just JEI and JER?

**Comment by Peca21** [comment]
> Does it work in a modpack with just JEI and JER?

Yes it does. So it must be mod compatibility issue?

**Comment by set92** [comment]
@Peca21 could you check if you have ice and fire mod and citadel? I think this issue is the same as #269, and they tracked the bug to those mods

**Comment by 04Slash** [comment]
> @Peca21 could you check if you have ice and fire mod and citadel? I think this issue is the same as #269, and they tracked the bug to those mods

I had Ice and Fire + Citadel installed during my testing (which was working as intended). Using versions `citadel-1.6.3` and `iceandfire-2.1.6-1.16.4`.

@Peca21 and @detecjack Are you two playing a modpack that can be downloaded? If so, I can look a bit further into it and try to find out why it's not working.

As for @set92 I tried loading Valhelsia 3 with no modifications except for updating both JEI (`jei-1.16.5-7.7.0.104`) and JER (`JustEnoughResources-1.16.5-0.12.1.121`) and it seemed to work. What other modifications did you make to the pack?

**Comment by Peca21** [comment]
I don't have citadel/ice and fire in my modpack. I am using curseforge for my modpack so I can export the modpack if you want.

**Comment by 04Slash** [comment]
Yea, if you don't mind I'll take a look at it and see if I can figure out why it's not working properly.

**Comment by Peca21** [comment]
https://www.dropbox.com/s/e6xqmkhxjb6gzh1/Preview2.rar?dl=1
This is the world I am playing on RN (803 MB)

https://www.dropbox.com/s/kenvb12yd63vcjx/Testing165.zip?dl=1
This is the exported modpack from curseforge (48 MB)

**Comment by 04Slash** [comment]
Sorry for the late replies. This kind of slipped my mind.

I've been able to narrow it down to `Draconic-Evolution-1.16.5-3.0.1.1-universal`. By the looks of it, DE is not officially released. I don't know when you compiled the mod but it hasn't been worked on in 21 days. If this was after you initially compiled the mod then you could try compiling it again and see if the changes fixed anything. Otherwise, you could try reporting it but with unreleased versions you may be telling the dev what they already know.

I have no idea if the above applies to the other people having this issue but I've removed just Draconic from what @Peca21 gave me and the issue was no longer present. If the others wish to supply a Curseforge export as Peca did, I'll gladly take a look at it to help out Way2muchnoise.

**Comment by Peca21** [comment]
Yes DE is not released, but on their discord Brandon is sharing early builds. Also this jar is indeed built from GitHub because the latest one from their discord is crashing.

**Comment by 04Slash** [comment]
Ah, I figured the mod was built from source by you. If the dev is releasing the mod themselves then I'd make an issue report on their issue tracker notifying them of this issue so it could be fixed before release. Or at the very least notify them on discord of the issue (or if they specified a method to report bugs, do that).

**Comment by set92** [comment]
@04Slash If you want to check it be my guest, the mod pack is a weird merge between Valhelsia 3 with the mods of ATM6 to have more content, so is quite big, and possibly the problem comes from there. I remove ice and fire and draconic evolution, but it doesn't show the chart for iron ore, and since is so big it takes few minutes to load all and is annoying to find which mod is the problem removing them one by one.

World:
https://www.dropbox.com/s/29m4mbbvwhce4c6/Ellura.zip?dl=1

Mod pack exported with MultiMC 5 (0.6.12-1530):
https://www.dropbox.com/s/zirfqorzlj0gj2c/Valhelsia%203%20-%201.16.5.zip?dl=1

**Comment by 04Slash** [comment]
@set92 Generally removing mods one by one is not a great idea. I do a binary search which is a lot faster. You start by removing half the mods. Then add in the missing library/core/whatever else mods. If the problem persists, remove half the remaining mods. If not, remove all mods and add the other half. Repeat this until you've narrowed it down to as few mods as possible. I like to keep folders which each "stage" so I can easily know which ones are fine and which ones are not. It can be quite time consuming depending on how things go. I got lucky in the one I did for Peca because I removed half, added the other half but removed just draconic because it was complaining about something and I didn't feel like figuring out why. Turns out, that was the mod with the issues. Probably saved a good 30-45mins :p

First, I'd recommend you update JEI to .106. I can't confirm this will work on any other version (you've currently got .96). The following situations will include the use of both `jei-1.16.5-7.7.0.106` and `JustEnoughResources-1.16.5-0.12.1.121`.

- The first instance can be replicated with `futurepack-1.16.4-31.2.6840`. 
- The second instance can be replicated with `hardcoredungeons-13` (+ it's dependencies `curios-forge-1.16.5-4.0.5.1` and `kotlinforforge-1.12.1-obf`). Removing HCD prevents the issue.
- There's at least one more instance but I'm done with doing binary searches for now. I'm fairly certain it's a conflict with 2 different mods, whether this be a standalone mod or a core mod + mod. I'm guessing one is near the top of the list and one near the bottom since I've split the pack in half and both halves work but the full pack, without the above 2 mods, doesn't work.

**Comment by 04Slash** [comment]
Did a bit more searching and the third instance is `Transport-1.16.5-3.6.0`, and `EpicFightMod-1.16.5-3.2.2` (+ it's dependencies `titanium-1.16.5-3.2.8.2` and `Patchouli-1.16.4-51`). Removing either Transport or EFM will prevent the issue.

After removing FP, HCD, Transport, and EFM, JER started working again.

@way2muchnoise Can you confirm if this is an issue with JER or if it's an issue with the listed mods? If it's the listed mods, I'll go ahead and create an issue report for each of them.

**Comment by amoooooo** [comment]
I assume my problem is the same as this, any update?

**Comment by PenumbraStar** [comment]
Appears to be no fix as of 7/25/2021, hopefully this issue is solved soon.

**Comment by trogarus** [comment]
Same issues here. Not able to view ore gen and other things in JEI. Latest version of both mods (JEI & JER) plus forge 1-16-5 36.2.2

I can also confirm that mods mentioned above, specifically Futurepack, Patchouli, and Titanium are in my modpack (trogarus skyblock bees adventure https://www.curseforge.com/minecraft/modpacks/trogarus-skyblock-bees-adventure).

Hopefully his somehow helps diagnose the issues behind this problem.

**Comment by trogarus** [comment]
As of most recent mod update, I can confirm that at least vanilla ores and such are fixed in this release and show y levels and etc again. Much of the dungeon loot stuff also shows, and even vanilla villager trades. Have not looked much into  modded items yet at this point.

**Comment by GordonHollow** [comment]
Mob drops totally don't show
With latest release JER and JEI - same bug

**Comment by Jaden99c** [comment]
I also have this issue. Mod is installed but JER isnt showing mob drops for any mob. JEI is working fine

**Comment by MrDj200** [comment]
Same issue here with
JEI: `jei-1.16.5-7.7.1.152` and `jei-1.16.5-7.7.1.137` (I tried updating to see if that would fix it
JER: `JustEnoughResources-1.16.5-0.12.1.128`

in the Create above and beyond modpack

**Comment by Doming3000** [comment]
Same problem, any solution?

**Comment by way2muchnoise** [comment]
Closing issue as I no longer support 1.16
